crates/utils/re_math/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "re_math"
+authors.workspace = true
+description = "3D math utilities built on glam"
+edition.workspace = true
+homepage.workspace = true
+include.workspace = true
+license.workspace = true
```

Comment:
we can't just apply our own license when it's clearly c&p from a different project to which other people contributed